unittests/unit/CMakeLists.txt

```diff
@@ -42,24 +38,22 @@ if(TARGET dart-utils)
   dart_add_test("unit" test_DartResourceRetriever)
   target_link_libraries(test_DartResourceRetriever dart-utils)
   
-  if(NOT MSVC)
-    dart_add_test("unit" test_FileInfoWorld)
-    target_link_libraries(test_FileInfoWorld dart-utils)
-  endif()
+  dart_add_test("unit" test_FileInfoWorld)
+  target_link_libraries(test_FileInfoWorld dart-utils)
 
   dart_add_test("unit" test_PackageResourceRetriever)
   target_link_libraries(test_PackageResourceRetriever dart-utils)
 
-  if(NOT MSVC)
-    dart_add_test("unit" test_MjcfParser)
-    target_link_libraries(test_MjcfParser dart-utils)
-    
-    dart_add_test("unit" test_SdfParser)
-    target_link_libraries(test_SdfParser dart-utils)
-    
-    dart_add_test("unit" test_SkelParser)
-    target_link_libraries(test_SkelParser dart-utils)
-    
+  dart_add_test("unit" test_MjcfParser)
+  target_link_libraries(test_MjcfParser dart-utils)
+  
+  dart_add_test("unit" test_SdfParser)
+  target_link_libraries(test_SdfParser dart-utils)
+  
+  dart_add_test("unit" test_SkelParser)
+  target_link_libraries(test_SkelParser dart-utils)
+
+  if(NOT MSVC) 
     dart_add_test("unit" test_VskParser)
     target_link_libraries(test_VskParser dart-utils)
   endif()
```

Comment:
Don't need to be fixed in this PR, but did you get any hint why `test_VskParser` doesn't build or pass with MSVC?

Reply:
I briefly saw a `0xC0000135` that typically means dll not found, so I guess it could be something related to dll of the dependencies not being in the path or not being copied in the directory with the test executables. In any case, I tried to reduce the modification as possible to have the smallest possible PR to ensure that it could be merged without side-effects.

Reply:
Thanks, let me take a look!